src/app/modules/model/test-navigator-tree-node.ts

```diff
@@ -15,15 +15,17 @@ export class TestNavigatorTreeNode implements TreeNode {
   leafCssClasses = '';
   cssClasses = '';
   expanded = undefined;
+  public  root: TestNavigatorTreeNode;
 
-  constructor(private workspaceElement: WorkspaceElement) {
+  constructor(private workspaceElement: WorkspaceElement, root: TestNavigatorTreeNode) {
```

Comment:
Default value?
```constructor(private workspaceElement: WorkspaceElement, root: TestNavigatorTreeNode = null)```
Does that make sense, or will that become an invalid way to invoke the constructor…?